<ui version="4.0" >
 <class>MultipartDocFormatConfiguratorWidget</class>
 <widget class="QWidget" name="MultipartDocFormatConfiguratorWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>75</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Create annotation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QRadioButton" name="separateRB" >
     <property name="text" >
      <string>Separate sequences</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QRadioButton" name="mergeRB" >
     <property name="text" >
      <string>Merge sequences</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="gapLabel" >
       <property name="text" >
        <string>Gap length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gapSpin" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>10000</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mergeRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>245</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel" >
     <x>284</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
